README.md

```diff
@@ -64,6 +64,8 @@ Thanks a lot to all the previous [contributors](https://github.com/i18next/i18ne
 - **-l, --locales <list>**: The locales in your applications. Defaults to `en,fr`
 - **--directoryFilter**: Globs of folders to filter
 - **--fileFilter**: Globs of files to filter
+- **--keep-removed**: Prevent keys no longer found from being removed
+- **--no-old**: Avoid saving the _old files
```

Comment:
I completely agree. Since _old file writing is the default, am I correct in assuming it would be disabled with `i18next --write-old=false`?
If this is your preference I'll fix it right away.